platformio.ini

```diff
@@ -285,6 +285,17 @@ build_flags =
 [env:tx-radiomaster-bandit-micro-900] ; Unified_ESP32_900_TX
 extends = env_common_tx_esp32
 board = esp32dev
+build_src_filter = 
+  ${env_common_tx_esp32.build_src_filter}
+  +<modules/sx12xx-lib/src/sx127x.cpp>
+  +<Common/thirdparty/gdisp.c>
+build_flags = 
+  ${env_common_tx_esp32.build_flags}
+  -D TX_ELRS_RADIOMASTER_BANDIT_MICRO_900_ESP32
+
+[env:tx-radiomaster-bandit-900] ; Unified_ESP32_900_TX
+extends = env_common_tx_esp32
+board = esp32dev
```

Comment:
the following lib_deps should be just two blanks ident

Reply:
Addressed